source/includes/APIReference/CustomFields/_create.md.erb

```diff
@@ -0,0 +1,40 @@
+## Create Custom Fields
+
+<%= partial "includes/APIReference/CustomFields/Examples/create.tmpl.erb" %>
+
+Add one or more customfields.
+
+### HTTP Request
+
+<img src="images/post.png" alt="post"/><api>https://rest.tsheets.com/api/v1/customfields</api>
+
+### Properties
+_Pass an array of customfield objects as the value to a 'data' property (see example)._
+
+|                |             |             |
+| -------------: | :---------: | ----------- |
+| **name**<br/>required | _String_ | The name of this customfield. |
+| **required**<br/>optional | _Boolean_ | If true, a non-null value must be selected for this customfield when it appears on a timesheet. |
+| **active**<br/>optional | _Boolean_ | If false, this `customfield` is considered archived and will not be visible or usable. |
+| **applies_to**<br/>required | _String_ | Allowed values: 'timesheet'. Specify that this `customfield` should appear on timesheets (additional customfield types may be supported in the future). |
+| **type**<br/>required | _String_ | Allowed values: 'managed-list', 'free-form'.  A managed-list customfield is intended to be displayed as a dropdown list, where each 'option' is a customfielditem. A 'free-form' customfield supports text entry and does not require a collection of customfielditems to accompany it. |
+| **show_to_all**<br/>optional | _Boolean_ | Default: `true`.  Declares whether this customfield should exist on timesheets for all workers. If false, it will only appear on timesheets for workers to whom it has been assigned. |
+| **short_code**<br/>optional | _String_ | This is a shortened code or alias that is associated with the customfield. It may only consist of letters and numbers.
+
+For a full list of the properties that may be set on a customfield, see [The Custom Field Object](#the-custom-field-object).
+
+### Status Codes
+Each customfield that is created will come back with a `_status_code` and `_status_message` that will indicate whether
+the customfield was created successfully. If there was a problem creating a customfield, there may also be an additional
+field, `_status_extra`, which will contain more details about the failure.
+
+|         |          |
+| :-----: | :------- |
+| <code class="level200">200</code> | OK. customfield was created successfully. |
```

Comment:
In the case of creating a customfield with a name that already is tied to an archived customfield, we actually do not create one, but instead unarchive the archived one.  I'm thinking we should give a status back of 202 in that case, similar to what we do for Geolocations.  

```
| <code class="level200">202</code> | OK. customfield was accepted, but not created (because we already have a customfield with that name. We've unarchived it for you.) |
```

Reply:
Did some manual testing and found that through the UI, it actual stops you from creating the name at all if it is in use, even if it is used by an archived custom field. The current implementation in the API was to rename the archived one with a date stamp appended at the end, and create the new one with the desired name. Is this difference acceptable or do we try to mirror functionality?

Reply:
Ok, I'm going to leave the docs as they are with the `200` response since the API will not provide the same side-effects that the web UI provides.